pkg/util/test/util.go

```diff
@@ -2547,7 +2550,8 @@ func validateCsiContainerInPxPods(namespace string, csi bool, timeout, interval
 				podsReady++
 			}
 
-			if pxVersion.LessThan(pxVer2_13) {
+			// TODO: Need to change "pxOperatorMasterVersion" to the release version when released
+			if opVersion, _ := GetPxOperatorVersion(); pxVersion.LessThan(pxVer2_13) && opVersion.Equal(pxOperatorMasterVersion) {
```

Comment:
I think we still need this validation, where are we validating CSI containers inside OCI-monitor pods? We only need a condition to not validate CSI containers inside API pods, if the feature is not in the operator version we are validating against.

Reply:
Oh okay got it, will make the changes